src/middleware.ts

```diff
@@ -20,23 +20,23 @@ export async function middleware(req: NextRequest) {
 
   // Ensure user is on the correct language in Landing
   // p.s. The other pages don’t matter because you land at Landing. :)
-  if (route == "/") {
-    // Detect browser language
-    const browserLang = req.headers
-      .get("accept-language")
-      ?.split(",")?.[0]
-      .split("-")?.[0];
-
-    // Redirect to the correct language
-
-    // From Thai to English
-    if (browserLang == "en" && locale != "en-US")
-      return NextResponse.redirect(new URL("/en-US", req.url));
-    // From Thai (or unsupported language) to Thai (which is the default
-    // language)
-    else if (browserLang != "en" && locale == "en-US")
-      return NextResponse.redirect(new URL("/", req.url));
-  }
+  // if (route == "/") {
+  //   // Detect browser language
+  //   const browserLang = req.headers
+  //     .get("accept-language")
+  //     ?.split(",")?.[0]
+  //     .split("-")?.[0];
+
+  //   // Redirect to the correct language
+
+  //   // From Thai to English
+  //   if (browserLang == "en" && locale != "en-US")
+  //     return NextResponse.redirect(new URL("/en-US", req.url));
+  //   // From Thai (or unsupported language) to Thai (which is the default
+  //   // language)
+  //   else if (browserLang != "en" && locale == "en-US")
+  //     return NextResponse.redirect(new URL("/", req.url));
+  // }
```

Comment:
You should add a comment saying why this is commented out. We’re gonna forget what this is and why it’s not used very soon.